flight/targets/pipxtreme/fw/pios_board.c

```diff
@@ -31,16 +31,57 @@
 #include <pios.h>
 #include <openpilot.h>
 #include <board_hw_defs.c>
+#include <modulesettings.h>
 #include <hwtaulink.h>
 #include <pios_hal.h>
 #include <rfm22bstatus.h>
 
+uintptr_t pios_com_telem_uart_bluetooth_id;
 #if defined(PIOS_INCLUDE_PPM)
 uintptr_t pios_ppm_rcvr_id;
 #endif
 
 uintptr_t pios_uavo_settings_fs_id;
 
+#define PIOS_COM_TELEM_RX_BUF_LEN 256
+#define PIOS_COM_TELEM_TX_BUF_LEN 256
+#define PIOS_COM_RFM22B_RF_RX_BUF_LEN 256
+#define PIOS_COM_RFM22B_RF_TX_BUF_LEN 256
+#define PIOS_COM_FRSKYSPORT_TX_BUF_LEN 16
+
+#if defined(PIOS_INCLUDE_USART) && defined(PIOS_INCLUDE_COM)
+static void PIOS_Board_configure_com (const struct pios_usart_cfg *usart_port_cfg, size_t rx_buf_len, size_t tx_buf_len,
```

Comment:
Sorry for the conflict that showed up here.  If rebased and this one is touched up I can run my tests on this and most likely merge Tuesday.